client/js/lounge.js

```diff
@@ -905,14 +905,11 @@ $(function() {
 						icon: "img/logo-64.png",
 						tag: target
 					});
-					notify.onclick = function() {
+					notify.addEventListener("click", function() {
```

Comment:
Is that necessary or just a nice-to-have? If the latter, why? Seems like there is nothing wrong with `onclick`, no?